src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        final int numberOfBalls = 3;
+        Ball[] balls = new Ball[numberOfBalls];
+        for (int i = 0; i < numberOfBalls; i++) {
+            balls[i] = new Lottery().getRandomBall();
```

Comment:
main can do with only one lottery instance, let's be efficient, what if it was costly operation